wintaser/AVIDumping/AudioConverterStream.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include <windows.h>
+#include <malloc.h>
+#include <algorithm>

```

Comment:
`<malloc.h>` and `<algorithm>` seem unneeded here (include in the .cpp instead), also perhaps `<cstdlib>` instead of non-standard `<malloc.h>`?